python/ray/tune/tune.py

```diff
@@ -166,14 +201,24 @@ def run(run_or_experiment,
             `num_samples` of times.
         local_dir (str): Local dir to save training results to.
             Defaults to ``~/ray_results``.
-        upload_dir (str): Optional URI to sync training results and checkpoints
-            to (e.g. ``s3://bucket`` or ``gs://bucket``).
-        trial_name_creator (Callable[[Trial], str]): Optional function
-            for generating the trial string representation.
-        trial_dirname_creator (Callable[[Trial], str]): Function
-            for generating the trial dirname. This function should take
-            in a Trial object and return a string representing the
-            name of the directory. The return value cannot be a path.
+        search_alg (Searcher): Search algorithm for optimization.
```

Comment:
The order of the arguments in the docstring should match the order of the arguments in the init

Reply:
done.